website/package.json

```diff
@@ -25,6 +25,7 @@
     "@emotion/styled": "^11.3.0",
     "@mdx-js/react": "^1.6.21",
     "@svgr/webpack": "^5.5.0",
+    "better-react-carousel": "^1.1.1",
```

Comment:
Any specific reason to use this package? Were there any alternatives you got chance to try upon? Reason for asking is that this package is:
- not well maintained
- 62kbs just for a carousel lib seems too much

Though, not a blocker; just something to think if we can check other alternatives and integrate for better flexibility and reusability.

Cc: @HarshCasper